### PR TITLE
REL-3110: Changed ephemerides directory for test ODBs and fixed credentials issue.

### PR DESCRIPTION
**Pull request by sraaphorst**

### CODE REVIEW

File: app/spdb/build.sbt [hunk 1]
```diff
@@ -573,6 +574,4 @@ def gsodb(version: Version) = AppConfig(
   bundles = List(
     BundleSpec(50, "edu.gemini.smartgcal.servlet", version)
   )
-) extending List(odbproduction(version), gsodbtest_credentials(version))
-
-
+) extending List(odbproduction(version), gsodb_credentials(version))
```
Comment (cquiroz) [inline app/spdb/build.sbt]: I know why you are making this change but somehow it worries me. How did it work correctly in the first place?

File: app/spdb/build.sbt [hunk 2]
```diff
@@ -516,14 +516,15 @@ def gnagsodb(version: Version) = AppConfig(
...
-    "edu.gemini.auxfile.fits.dest"         -> "/gemsoft/var/data/ictd/GN@SEMESTER@/@PROG_ID@",
-    "edu.gemini.auxfile.fits.host"         -> "gnconfig.gemini.edu",
-    "edu.gemini.auxfile.other.dest"        -> "/gemsoft/var/data/finder/GNqueue/Finders/@SEMESTER@/@PROG_ID@",
-    "edu.gemini.auxfile.root"              -> "/home/software/.auxfile",
-    "edu.gemini.dataman.gsa.summit.host"   -> "fits.hi.gemini.edu",
-    "edu.gemini.oodb.mail.smtpHost"        -> "smtp.hi.gemini.edu",
-    "edu.gemini.util.trpc.name"            -> "Gemini North AGS ODB",
-    "osgi.shell.telnet.ip"                 -> "10.1.46.11"
+    "edu.gemini.dbTools.tcs.ephemeris.directory" -> "/home/software/.ephemerides",
+    "edu.gemini.auxfile.fits.dest"               -> "/gemsoft/var/data/ictd/GN@SEMESTER@/@PROG_ID@",
```
Comment (cquiroz) [inline app/spdb/build.sbt]: Maybe we should change these to be local too

**Comment by cquiroz** [comment]
Thanks for this, it was surprising to stumble upon this and overwrite production files.

Could you, once merged in develop, also retarget it for `release-2017A`?

**Comment by sraaphorst** [comment]
@cquiroz I created another PR for release-2017A 😸

File: app/spdb/build.sbt (hunk 2 above)
Comment (sraaphorst) [inline app/spdb/build.sbt]: Agreed. I'll go ahead and change that, too.

File: app/spdb/build.sbt (hunk 1 above)
Comment (sraaphorst) [inline app/spdb/build.sbt]: I have no idea at all... I'll investigate a bit further since merging this PR is not an emergency priority.

Comment (tpolecat) [inline app/spdb/build.sbt]: Yeah it would be good to know why it worked before.

**Comment by swalker2m** [comment]
What is the current status of this?  It's been hanging around for a while now but it seems like it could be merged.  So the production ODB will continue to write ephemeris files to the same place, but the test ODBs will write them to a local directory? 

How are the ephemeris updates be used in testing?  Is there a test version of the TC[CS] that looks in the new test directory?

**Comment by sraaphorst** [comment]
This should all be done now by REL-3140 via the following PR:
https://github.com/gemini-hlsw/ocs/commit/a68451d60150f6d7d03416b5569a7a3aeed74d6f

There is no need for this PR anymore, which should be considered obsolete as I incorporated the changes here while making the changes to move the test ODBs.

I'll close this.

**Pull request closed without merging**